build.sbt

```diff
@@ -413,14 +413,27 @@ lazy val ratSettings = Seq(
 )
 
 lazy val unidocSettings = Seq(
-  ScalaUnidoc / unidoc / unidocProjectFilter := inProjects(sapi, udf),
+  ScalaUnidoc / unidoc / unidocProjectFilter := inProjects(sapi, udf, runtime1),
   ScalaUnidoc / unidoc / scalacOptions := Seq(
     "-doc-title",
     "Apache Daffodil " + version.value + " Scala API",
     "-doc-root-content",
     (sapi / baseDirectory).value + "/root-doc.txt"
   ),
-  JavaUnidoc / unidoc / unidocProjectFilter := inProjects(japi, udf),
+  ScalaUnidoc / unidoc / unidocAllSources :=
+    (ScalaUnidoc / unidoc / unidocAllSources).value.map { sources =>
+      sources.filter { source =>
+        //
+        // This filter limits the scaladoc to only files which have
+        // api, sapi, or japi as directory names in their paths.
+        //
+        val str = source.toString
+        str.contains("/api/") ||
```

Comment:
For reference, these are the new classes in the API:

```
root
  org
   apache
     daffodil
       runtime1
         api
           ChoiceMetadata
           ComplexElementMetadata
           DFDL
           DFDLPrimType
           ElementMetadata
           InfosetArray
           InfosetComplexElement
           InfosetDocument
           InfosetElement
           InfosetItem
           InfosetSimpleElement
           InfosetTypeException
           Metadata
           MetadataHandler
           ModelGroupMetadata
           SequenceMetadata
           SimpleElementMetadata
           TermMetadata
         layers 
           api
             ChecksumLayer
             Layer
```
I think all the Metadata ones are new public APIs (are they considered experimental or are we officially supporting them an requiring backwards compatibility?).

But I consider `DFDL` to be an internal API that people shouldn't use, at least until we get rid of SAPI/JAPI with DAFFODIL-1747. Im not sure about the others. Are any of these considered internal and should be filtered out?